management-console/README.md

```diff
@@ -28,12 +28,12 @@ create the runnable `.jar`.
 To define the needed parts of Thorntail, some dependencies are added
```

Comment:
Few lines above, there's one more occurence of `<groupid>`, wanna fix that too? :-)